estate/models/estate_property.py

```diff
@@ -0,0 +1,19 @@
+from odoo import models, fields
+
+class RealEstateproperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection([ ('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West') ])
```

Comment:
You can use Ruff for formatting if you're using VsCode
```suggestion
    garden_orientation = fields.Selection([('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')])
```